actions/host_get_id.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: host_get_id
+pack: zabbix
+runner_type: python-script
+description: Get the ID of a Zabbix Host
+enabled: true
+entry_point: host_get_id.py
+parameters:
+    host:
+        type: string
+        description: "Hostname of FQDN of the Zabbix Host"
```

Comment:
'Hostname of FQDN of the Zabbix Host' doesn't make much sense to me?

Is it hostname (like first component) or the FQDN?